test/util/test_runners/test_runner_args.py

```diff
@@ -19,7 +20,7 @@ class TestRunnerArgs:
     DTO to hold the tests environment arguments
     """
 
-    def __init__(self, run_oss_tests, url, account, login, password, invoke_cli_as_process,
+    def __init__(self, run_oss_tests, url, account, login, password,
```

Comment:
I think this work should be in a separate PR. The PR is quite big already. Removing the process flow also means we have to change all tests from `integration_test(true)` back to `@integration_test`. Change `integration_test_case.py` 

```
        if self.environment.invoke_process:
            return invoke_cli_as_process(self, *args, exit_code=exit_code)
        return invoke_cli_as_code(self, *args, exit_code=exit_code)
```
to just `return invoke_cli_as_code(self, *args, exit_code=exit_code)` etc etc